tools/openblas_support.py

```diff
@@ -120,7 +121,8 @@ def download_openblas(target, plat, ilp64):
 
     if not suffix:
         return None
-    filename = f'{BASEURL}/openblas{fnsuffix}-{OPENBLAS_LONG}-{suffix}'
+    BASEURL = f'{BASE_LOC}/{openblas_version}/download'
+    filename = f'{BASEURL}/openblas{fnsuffix}-{openblas_version}-{suffix}'
```

Comment:
Nit: As you noticed, BASEURL is not really a global (capitalized) variable, this should be styled differently now, and the `BASEURL` all along should have had a `_` between the words. Maybe get rid of it altogether?

Reply:
Removed it altogether.